src/main/java/com/sda/entity/Category.java

```diff
@@ -0,0 +1,62 @@
+package com.sda.entity;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "Category")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "ID", unique = true, nullable = false)
+    private Long id;
+
+    @Column(name = "NAME", unique = true, nullable = false)
```

Comment:
Maybe it is better to provide name length as well, because default value is 255.